src/benchmark.zig

```diff
@@ -133,12 +89,40 @@ pub fn main() !void {
     );
     defer client.deinit(allocator);
 
-    try send(
-        &io,
-        &client,
-        .create_accounts,
-        std.mem.sliceAsBytes(accounts[0..]),
-    );
+    // Init accounts.
+    var batch_accounts = try std.ArrayList(tb.Account).initCapacity(allocator, account_count_per_batch);
+    var account_index: usize = 0;
+    while (account_index < account_count) {
+        // Fill batch.
+        while (account_index < account_count and
+            batch_accounts.items.len < account_count_per_batch)
+        {
+            batch_accounts.appendAssumeCapacity(.{
+                .id = account_index + 1,
```

Comment:
Can you do the `bitReverse` trick on account ids too? That will give more realistic `{debit,credit}_account_id` index performance.

Reply:
Not much change in perf, surprisingly.